packages/components/src/SubHeaderBar/TitleSubHeader/TitleSubHeader.scss

```diff
@@ -65,6 +65,14 @@ $tc-input-subheader-size-medium: 1.4rem !default;
 			}
 		}
 
+		&:not(:hover) {
```

Comment:
I would prefer to have a `display: none;` by default (or an aria-hidden or whatever)